server/src/com/treode/server/package.scala

```diff
@@ -48,6 +48,13 @@ package object server {
       rsp
     }
 
+    def stale (req: Request, time: TxClock): Response = {
```

Comment:
I suspect the method `clear` below isn't used anywhere. Please see if you can remove it.